web/src/app/rotations/RotationUserList.js

```diff
@@ -50,7 +50,7 @@ export default class RotationUserList extends React.PureComponent {
     return (
       <React.Fragment>
         <Card>
-          <CardHeader title='Users' />
+          <CardHeader style={{ margin: 0 }} component='h3' title='Users ' />
```

Comment:
when adding new styles we prefer to use `className` with mui styles to be consistent (styles can always be found at the top of the file)